src/main/scala/is/hail/distributedmatrix/BlockMatrix.scala

```diff
@@ -593,17 +593,15 @@ class BlockMatrix(val blocks: RDD[((Int, Int), BDM[Double])],
       .map { case (i, a) => IndexedRow(i, new DenseVector(a)) },
       nRows, nCols)
   }
-    
-  // rowsToKeep is an array of distinct row indices
-  def filterRows(rowsToKeep: Array[Long]): BlockMatrix = this.transpose().filterCols(rowsToKeep).transpose()
+  
+  def filterRows(keep: Array[Long]): BlockMatrix = this.transpose().filterCols(keep).transpose()
```

Comment:
This doesn't seem necessary.  Moreover, it seemed clearer the old way.  Don't change it again, but FYI.